src/pages/rewards/[address]/index.tsx

```diff
@@ -225,10 +289,64 @@ const Rewards: React.FC = () => {
                 void notify(farm, toWei(amount));
                 refresh();
               }}
-              mr={2}
+              mr={1}
             >
               Notify custom {lookup.rewardTokenSymbol}
             </Button>
+
+            <Button
+              onClick={() => {
+                const reward_dist_address = prompt(
+                  `Enter the address of the rewards distribution`
+                );
+
+                if (!reward_dist_address) {
+                  console.warn("Invalid address");
+                  return;
+                }
+                void setRewardsDistribution(farm, reward_dist_address);
+              }}
+              mr={1}
+            >
+              Set rewards distribution {lookup.rewardTokenSymbol}
+            </Button>
+
+            <Button
+              onClick={() => {
+                const input = prompt(`Enter the address of the new owner`);
+
+                if (!input) {
+                  console.warn("Invalid address");
+                  return;
+                }
+
+                void nominateNewOwner(farm, input);
+                refresh();
+              }}
+              mr={1}
+            >
+              Nominate owner
+            </Button>
+
+            <Button
```

Comment:
Would be good to check if the connected wallet is the nominated owner. If they are not, hide or disable this button